projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.spec.ts

```diff
@@ -104,4 +113,36 @@ describe('SearchAiResultsComponent', () => {
       expect(component.hasError).toBe(true);
     });
   });
+
+  describe('query params change', () => {
+    beforeEach(() => {
+      searchAiService = TestBed.inject(SearchAiService);
+    });
+
+    it('displays skeleton when loading is true', () => {
+      mockQueryParams.next({ query: 'test', agentId: 'agentId1' });
+      component.performAiSearch();
+      expect(component.loading).toBe(true);
```

Comment:
can you change all toBe(true) and toBe(false) to toBeTrue() and toBeFalse() in your tests?